core/src/main/java/org/ehcache/CacheManagerBuilder.java

```diff
@@ -90,4 +92,8 @@ T cast(EhcacheManager ehcacheManager) {
   public static CacheManagerBuilder<CacheManager> newCacheManagerBuilder() {
     return new CacheManagerBuilder<CacheManager>();
   }
+
+  public static PersistenceConfiguration persistence(String location) {
```

Comment:
I personally would prefer to have File as the argument type of the persistence() method